docs/operating-procedures.md

```diff
@@ -80,9 +80,9 @@ objects:
 
     $ /opt/omero/server/OMERO.server/bin/omero login public@localhost -w public
     $ /opt/omero/server/OMERO.server/bin/omero hql --limit -1 --ids-only --style csv 'select MIN(field.image.id) FROM WellSample AS field GROUP BY field.well.plate' > plates.txt
-    $ /opt/omero/server/OMERO.server/bin/omero hql --limit -1 --ids-only --style csv 'select d.id from Dataset d' > datases.txt
-    $ cat plates.txt | cut -d ',' -f2 | sed -e 's/^/Image:/' > ids.txt 
-    $ cat datasets.txt | cut -d ',' -f2 | sed -e 's/^/Dataset:/' >> ids.txt
+    $ /opt/omero/server/OMERO.server/bin/omero hql --limit -1 --ids-only --style csv 'select d.id from Dataset d' > datasets.txt
+    $ < plates.txt cut -d ',' -f2 | sed -e 's/^/Image:/' > ids.txt
```

Comment:
Since you're removing the extraneous `cat` you could also remove the `<` by passing the file as a cut argument `cut -d ',' -f2 <file>` 😀

Reply:
True, heh. I kind of like it as it is, the pipeline looks more in order with the input file at the start!

Reply:
I defer to the more active data importers, but I note remember are using the `$ < ...` idiom anywhere else.

Reply:
I vote for keeping it simple, bearing in mind we've got several data importers who may not be familiar with advanced shell syntax

Reply:
I'd thought what I'd written was clearer and simpler but will adjust. :laughing: